Classes/Controller/EntityFactController.php

```diff
@@ -89,6 +89,13 @@ public function showAction()
             }
         }
 
+        //add in default Icon if needed
+        foreach ($sameAsArray as $key => &$item) {
+            if(!$item['collection']['icon'])
+            {
+                $item['collection']['icon'] = "./typo3conf/ext/slub_entityfacts/Resources/Public/Icons/default_external_link.png";
```

Comment:
```suggestion
            if(!$item['collection']['icon']) {
                $item['collection']['icon'] = "./typo3conf/ext/slub_entityfacts/Resources/Public/Icons/default_external_link.png";
```

Please follow `[PSR-2 coding style](https://www.php-fig.org/psr/psr-2/)`:
_* Opening braces for control structures MUST go on the same line, and closing braces MUST go on the next line after the body._